sysutils/dmidecode/src/www/widgets/include/dmidecode.inc

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$dmidecode_title = gettext('System hardware information');
```

Comment:
I would add a hint that this is the dmidecode plugin.

Reply:
how?

Reply:
Just an example:

```suggestion
$dmidecode_title = gettext('System hardware information (dmidecode)');
```

The reason is that the plugin will show up this way in the list of widgets and if someone installs dmidecode, he will probably look for that in the list.

(also not a must have)